ppo.ipynb

```diff
@@ -208,34 +257,28 @@
     "        # Make the first observation.\n",
     "        agent.observe_first(timestep)\n",
     "\n",
-    "        pbar_rollout = tq.tqdm(range(len_rollout), position=1, leave=False)\n",
+    "        pbar_rollout = tq.tqdm(range(max_rollout), position=1, leave=False)\n",
     "        pbar_rollout.set_description(f\"Rollout step\")\n",
     "        for rollout_step in pbar_rollout:\n",
     "            pbar_rollout.set_postfix(reward=timestep.reward)\n",
     "            value = agent.get_value(timestep.observation)\n",
     "            action, log_prob = agent.select_action_and_prob(timestep.observation)\n",
     "            timestep = environment.step(action)\n",
     "            agent.observe(value, log_prob, action, timestep)\n",
-    "\n",
     "            rollout_return += timestep.reward\n",
     "            # Incrementing count of time step\n",
     "            t += 1\n",
     "            # Avg reward per global timestep\n",
     "            avg_reward += (timestep.reward - avg_reward) / t\n",
     "            # Add avg reward\n",
     "            writer.scalar(\"return/avg_per_timestep\", avg_reward, t)\n",
-    "\n",
-    "            # Have the agent observe the timestep and let the agent update itself.\n",
     "            if timestep.last():\n",
-    "                timestep = environment.reset()\n",
-    "                agent.observe_first(timestep)\n",
-    "                all_returns.append(rollout_return)\n",
-    "                rollout_return = 0\n",
+    "                break\n",
```

Comment:
The problem is that the progress bar stays in the prints. I think we should find a work around so that the sub progress bar disappears. Maybe a "while" loop without a progress bar.